src/main/java/org/mifos/processor/bulk/camel/routes/AccountLookupRoute.java

```diff
@@ -0,0 +1,46 @@
+package org.mifos.processor.bulk.camel.routes;
+
+import org.apache.camel.Exchange;
+import org.apache.camel.LoggingLevel;
+import org.mifos.processor.bulk.camel.config.CamelProperties;
+import org.mifos.processor.bulk.camel.processor.AccountLookupCallbackProcessor;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Component;
+
+import static org.mifos.processor.bulk.camel.config.CamelProperties.*;
+import static org.mifos.processor.bulk.zeebe.ZeebeVariables.*;
+
+@Component
+public class AccountLookupRoute extends BaseRouteBuilder{
+    @Autowired
+    AccountLookupCallbackProcessor accountLookupCallbackProcessor;
+    @Override
+    public void configure() throws Exception {
+
+        from("rest:POST:/accountLookup/Callback")
+                .log(LoggingLevel.DEBUG, "######## -> ACCOUNT LOOKUP CALLBACK")
+                .process(accountLookupCallbackProcessor)
+                .setBody(constant("Received"))
+                .setHeader(Exchange.HTTP_RESPONSE_CODE, constant(200))
+                .end();
+
+        from("direct:send-account-lookup")
+                .id("account-lookup")
+                .process(exchange -> {
+                    String callbackUrl = exchange.getProperty(CALLBACK, String.class);
+                    exchange.getIn().setHeader(CALLBACK, callbackUrl);
+                })
+                .setHeader(Exchange.HTTP_METHOD, constant("GET"))
+                .setHeader(
+                        Exchange.HTTP_QUERY,
+                        simple(
+                                PAYEE_IDENTITY + "=${exchangeProperty." + PAYEE_IDENTITY + "}&" +
```

Comment:
donot use + for concatenation, try string builder instead.

new Stringbuilder().append("").append("")....toString()